seed/validate.go

```diff
@@ -78,3 +79,14 @@ func ValidateFromYaml(seedYamlFile string) error {
 
 	return nil
 }
+
+var validSeedSystemLabel = regexp.MustCompile("^[a-zA-Z0-9][a-zA-Z0-9-]+$")
```

Comment:
DoYoKnOwBooGiEWooGie?

Reply:
@zyga stop sharing my ubuntu core backup labels on the internet :stuck_out_tongue:

Reply:
Lol :D